frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -41,6 +45,11 @@ export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
   });
 
   React.useEffect(() => {
+    const namespace = getFilterValueByNameFromURL('namespace', location);
+    if (namespace.length > 0) {
+      dispatch(setNamespaceFilter(namespace));
+      setShowFilters(true);
```

Comment:
This will show the search filter as well, on pages that may not even have the namespace filter. So you need to check for whether the `noNamespaceFilter` is set.